Tests/QueryTests.swift

```diff
@@ -0,0 +1,343 @@
+//
+//  QueryTests.swift
+//  Contentful
+//
+//  Created by JP Wright on 06/03/2017.
+//  Copyright © 2017 Contentful GmbH. All rights reserved.
+//
+
+@testable import Contentful
+import XCTest
+import Nimble
+import DVR
+import Interstellar
+import CoreData
+
+
+// PERSISTENCE
+//final class Cat: NSManagedObject, ContentModel {
```

Comment:
Are those tests to be used once persistence is updated, or can we uncomment them, make them useful?